3.0.0.md

```diff
@@ -0,0 +1,209 @@
+# Upgrading Parse Server to version 3.0.0
+
```

Comment:
the elimination of `Parse.Promise` should be mentioned in the opening sentence and then covered with a paragraph.  my suggestion:

With the 3.0.0, parse-server has completely revamped it's cloud code interface. Gone are the backbone style calls and Parse.Promise, welcome native promises and async / await.

3.0.0 drops Parse.Promise.  References to `Parse.Promise` will need to be removed and the native promise API adhered to.  See [below]('migrating-promises`) for some examples:

.....

I think we'd want a section with known issues:

Parse.Promise.as('ok') => Promise.resolve('ok').

const { height, width  } = await Parse.Promise.when(getHeight(), getWidth());   
const [height, width] = Promise.all([getHeight(), getWidth()]);

Parse.Promise.error('uh-oh')  => Promise.reject('uh-oh')

For Parse.Promise.is(), see: https://stackoverflow.com/questions/27746304/how-do-i-tell-if-an-object-is-a-promise for a discussion of what this means and suggestions that will fit your situation.

Parse.Promise.always() => Promise.finally()

--I'm not sure what Parse.Promise.done() is and if it is just an alias???--

Parse.Promise.fail() => Promise.catch()

Also note that any 'private' function on Parse.Promise (_continueWhile, _continueWith, or _thenRunCallbacks) will also need to be replaced.

Reply:
All of that are part of the JS SDK 2.0, I'll add a link to the JS SDK 2.0 migration guide in here if that's fine with you. I don't want to duplicate in both repos.

Reply:
i think you can just ignore this while long comment.  yes, its part of sdk migration.  doh.  look forward to reading @dplewis blog post ;) which is what should combine the two.